modules/health_quest/app/services/health_quest/appointment_service.rb

```diff
@@ -17,11 +17,13 @@ def get_appointments(start_date, end_date, pagination_params = {})
       end
     end
 
-    def get_appointment_by_id(id)
+    def get_appointment_by_id(_id)
       with_monitoring do
-        response = perform(:get, "#{get_appointments_base_url}/#{id}", {}, headers, timeout: 55)
+        response =
+          YAML.load_file(Rails.root.join(*appt_file)).with_indifferent_access
```

Comment:
Why are we no longer pulling data from an endpoint and instead pulling from a local yml?

I wouldn't think we would want to do this in production, even if the code is behind a feature flag.

Reply:
Good question. The endpoint to pull data from local/dev/playground does not exist since VISTA is not available in those environments. The MAS service in staging will support retrieving appointment data in the future but doesn't do so currently. The team responsible for the MAS service is currently working on letting vets-api hit it's appointment endpoints. We're hoping that those endpoints will be available in the next few sprints.

Until this endpoint to communicate with the service is available I'm mocking the data locally in the vets-api. I'm open to suggestions.